CHANGELOG.md

```diff
@@ -213,6 +213,7 @@ Additions to existing modules
   ⁻¹-selfInverse  : SelfInverse _⁻¹
   \\≗flip-//⇒comm : (∀ x y → x \\ y ≈ y // x) → Commutative _∙_
   comm⇒\\≗flip-// : Commutative _∙_ → ∀ x y → x \\ y ≈ y // x
+  ¹-anti-homo‿- : (x - y) ⁻¹ ≈ y - x
```

Comment:
```suggestion
  ⁻¹-anti-homo‿- : (x - y) ⁻¹ ≈ y - x
```

Reply:
Suggested change: thus Jemma as stated moves to `Algebra.Properties.AbelianGroup`, as a renaming of a correctly stated lemma in `...Group`?